src/components/NotificationCenter.tsx

```
@@ -0,0 +1,641 @@
+  priority: NotificationPriority;
+  title: string;
+  message: string;
+  timestamp: number;
+  read: boolean;
+  persistent?: boolean;
+  actionLabel?: string;
+  actionUrl?: string;
+  onAction?: () => void;
+  autoHide?: boolean;
+  hideDelay?: number;
+  metadata?: Record<string, any>;
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)